test/autorest/generated/urlgroup/internal/urlgroup/paths.go

```diff
@@ -96,7 +96,7 @@ func (PathsOperations) ByteNullCreateRequest(u url.URL, bytePath []byte) (*azcor
 
 // ByteNullHandleResponse handles the ByteNull response.
 func (PathsOperations) ByteNullHandleResponse(resp *azcore.Response) (*PathsByteNullResponse, error) {
-	if !resp.HasStatusCode(http.StatusOK) {
+	if !resp.HasStatusCode(http.StatusBadRequest) {
```

Comment:
We're expecting null scenarios to return a BadRequest status?

Reply:
I know this comes from the swagger, but I'm wondering what changed. Also, shouldnt it be without the ! for bad requests? That would be an error wouldnt it?

Reply:
The way it's worded in the test, the operation returns invalid data that should cause unmarshalling to fail so the API call returns an error.  So if you actually get back the 400 and no error something went wrong elsewhere (it's kinda weird).

Reply:
As for what changed, this was fixed via `formatStatusCodes()`.  Before that we had a hard-coded `http.StatusOK`.

Reply:
Hmmm it is very strange. But I guess this is the expected output then. Doesnt sound like real world scenarios should work like this though.